torch/_library/custom_ops.py

```diff
@@ -468,3 +470,11 @@ def check(arg):
         yield from check(arg)
     for kwarg in kwargs.values():
         yield from check(kwarg)
+
+
+def _maybe_get_opdef(qualname):
```

Comment:
qualname sounds like it should be a str object which is not true given the below...

Reply:
Also it's used only in one place, should we just move it to that other file?

Reply:
> qualname sounds like it should be a str object which is not true given the below...

Ya

> Also it's used only in one place, should we just move it to that other file?

I'm trying to not pollute library.py too much